apps/web/package.json

```diff
@@ -11,7 +11,7 @@
     "format:fix": "prettier . --write --cache --cache-location=\"node_modules/.cache/prettiercache\" --log-level=warn",
     "lint": "eslint . --cache --cache-location \"node_modules/.cache/.eslintcache\" --max-warnings 0",
     "lint:fix": "eslint . --fix --cache --cache-location \"node_modules/.cache/.eslintcache\" --max-warnings 0",
-    "start": "next start",
+    "start": "dotenv -e ../../.env next start",
```

Comment:
That's basically an alias to the `dev` script?

Reply:
if i run build i cant run start cause it wont pickup the env vars

Reply:
dev is specifically for dev (not prod build)

Reply:
oh yeah I'm so dumb